readme.md

```diff
@@ -167,9 +165,33 @@ You can programmatically access Favicons configuration (icon filenames, HTML, ma
 import { config } from "favicons";
 ```
 
-## Output
+Below you will find a simple working example to generate an output. Amend the `src`, `dest`, `htmlBasename` and `configuration` constants to suit your own needs.
+
+```js
+import favicons from 'favicons';
+import fs from 'fs/promises';
+import path from 'path';
+
+const src = './icon.svg'; // Icon source file path.
+const dest = './favicons'; // Output directory path.
+const htmlBasename = 'index.html'; // HTML file basename.
 
-For the full list of files, check `config/files.json`. For the full HTML code, check `config/html.js`. Finally, for the full list of icons, check `config/icons.json`.
+// Configuration (see above in the README file).
+const configuration = {
+  path: '/favicons',
+  appName: 'My Great App',
+  appShortName: 'Great App',
+  appDescription: 'A great application to test itgalaxy/favicons.',
+  // Extra options...
+};
+
+// Below is the processing.
+const response = await favicons(src, configuration);
+await fs.mkdir(dest, {recursive: true, });
+response.images.forEach((image) => fs.writeFile(path.join(dest, image.name), image.contents));
```

Comment:
you need to `await` for promises. Use either `Promise.all` or a `for-of` loop.

Reply:
If you do not `await` for a promise, you cannot catch any errors happening.

Assuming that this code is what users of `favicons` will copy and use we need to be very careful with what we recommend. There is no guarantee that files are written until promises are awaited. see https://typescript-eslint.io/rules/no-floating-promises/